### PR TITLE
sync footer links & update sponsors block

### PR DESCRIPTION
**Pull request by Arhell**
fix issue https://github.com/cakephp/plugins.cakephp.org/issues/92

### CODE REVIEW

File: config/site.php [hunk 1]
```diff
@@ -140,11 +140,11 @@
                 ],
                 'documentation' => [
                     'book' => [
-                        'url' => 'http://book.cakephp.org/',
+                        'url' => 'http://book.cakephp.org/4',
```
Comment (ADmad) [inline config/site.php:143]: There's no need to change this. The manual homepage will always be for current major release. Just the protocol should be changed to https.

File: config/site.php [hunk 2]
```diff
@@ -69,6 +54,26 @@
                         'options' => ['target' => '_blank'],
                         'title' => __('CakeFest'),
                     ],
+                    'newsletter' => [
+                        'url' => [
```
Comment (ADmad) [inline config/site.php:58]: This should be a string URL https://cakephp.org/pages/newsletter, since it's from another domain.

**Review by ADmad — CHANGES_REQUESTED** [review]

File: config/site.php (hunk 1 above)
Comment (Arhell) [reply config/site.php]: Done

File: config/site.php (hunk 2 above)
Comment (Arhell) [reply config/site.php]: Done

File: config/site.php [hunk 3]
```diff
@@ -156,6 +151,10 @@
                         ],
                         'title' => __('Videos'),
                     ],
+                    'security' => [
+                        'url' => 'https://book.cakephp.org/4.0/en/contributing/tickets.html#reporting-security-issues',
```
Comment (ADmad) [inline config/site.php:155]: This link would be more permanent https://github.com/cakephp/cakephp#security

Comment (Arhell) [reply config/site.php]: Done

File: config/site.php [hunk 4]
```diff
@@ -8,36 +8,21 @@
         'menu' => [
             'items' => [
                 'community' => [
-                    'team' => [
+                    'getInvolved' => [
                         'url' => [
```
Comment (ADmad) [inline config/site.php:12]: This should also be a static URL, sorry I didn't notice earlier.

Comment (Arhell) [reply config/site.php]: Done
Changed all links

**Review by ADmad — APPROVED** [review]

**Pull request merged**